cassandra/1.10/README.md

```diff
@@ -0,0 +1,369 @@
+# How to use Apache Cassandra on DC/OS
+
+[Apache Cassandra](https://cassandra.apache.org/) is a distributed, structured storage system. Cassandra clusters are highly available, scalable, performant, and fault tolerant. DC/OS Cassandra allows you to quickly configure, install, and manage Apache Cassandra. Multiple Cassandra clusters can also be installed on DC/OS and managed independently, so you can offer Cassandra as a managed service to your organization.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Anyone who wants to deploy a distributed database on DC/OS. Beginner level.
+- Scope:
+ - Install the DC/OS Cassandra service.
+ - Use the enhanced DC/OS CLI operations for Cassandra.
+ - Validate that the service is up and running.
+ - Connect to Cassandra and perform CRUD operations.
+
+**Terminology**:
+
+- **Node**: A running Cassandra instance.
+- **Cluster**: Two or more Cassandra instances that communicate over gossip protocol.
+- **Keyspace**: A namespace that defines how data is replicated on nodes.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Cassandra](#install-cassandra)
+  - [Typical installation](#typical-installation)
+  - [Custom manual installation](#custom-manual-installation)
+  - [Validate installation](#validate-installation)
+- [CRUD operations](#perform-crud-operations)
+- [Uninstall](#uninstall)
+
+## Prerequisites
+
+- A running DC/OS 1.10 cluster with 3 nodes each with 1.5 CPU shares, 5376MB of memory and 11264MB of disk for running Cassandra nodes and 1 node with 0.5 CPU shares, 2048MB of memory for running the service scheduler.
+- [DC/OS CLI](https://dcos.io/docs/1.10/cli/install/) installed.
+
+## Install Cassandra
+
+Assuming you have a DC/OS cluster up and running, the first step is to [install Cassandra](https://docs.mesosphere.com/manage-service/cassandra/).
```

Comment:
This link is broken, I think it should link to https://docs.mesosphere.com/service-docs/cassandra ?